lib/modules/auth/login/bloc/login_state.dart

```diff
@@ -0,0 +1,36 @@
+import '/models/login/login_response_model.dart';
+
+abstract class LoginState {
+  const LoginState();
+}
+
+class LoginInitialState extends LoginState {
+  const LoginInitialState();
+}
+
+class LoginLoadingState extends LoginState {
+  const LoginLoadingState();
+}
+
+class LoginSuccessState extends LoginState {
+  final LoginResponseModel model;
+
+  const LoginSuccessState(this.model);
+}
+
```

Comment:
LoginErrorState comment how you use it

Reply:
When an error occurs return LoginErrorState with the message from exception handler class